app/mutations/industry/editUserMutation.ts

```diff
@@ -1,19 +1,38 @@
 import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
 import BaseMutation from '../BaseMutation';
+import {
+  editUserMutation as editUserMutationType,
+  editUserMutationVariables
+} from '../../__generated__/editUserMutation.graphql';
```

Comment:
```suggestion
} from 'editUserMutation.graphql';
```
The typescript server will look for files in the `__generated__` folder, no need to add the full relative path